packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -2717,6 +2717,24 @@ void main() {
     semantics.dispose();
   });
 
+  // TODO(chunhtai): remove this test once we resolve
+  //  https://github.com/flutter/flutter/issues/79818
```

Comment:
Nit: Period at the end for consistency since you had a period in your first comment above.